app/views/admin/worldwide_offices/index.html.erb

```diff
@@ -1,8 +1,4 @@
-<% content_for :back_link do %>
-  <%= render "govuk_publishing_components/components/back_link", {
-    href: admin_worldwide_organisations_path,
-  } %>
-<% end %>
+
```

Comment:
Can these blank lines be removed so that we're starting content on line 1 rather than line 2?